gammapy/scripts/image_pipe.py

```diff
@@ -68,12 +85,42 @@ def make_counts(self, obs_id):
         counts_map.data = counts_map.data.value
         self.maps["counts"] = counts_map
 
+    def _low_counts(self, obs_id):
```

Comment:
Based on the name, I would have expected `_low_counts` to return `True` if the counts are low.
It's currently the opposite.

Can you change True <-> False ?
Or, if you don't want that, make the name clearer?